model/item.cc

```diff
@@ -129,14 +129,21 @@ namespace minsky
       {
         if (hypot(x-p->x(), y-p->y()) < portRadius*zoomFactor())
           return ClickType::onPort;
-      }
+      }          
+     
+    // then, check whether a resize handle has been selected  
+	double w=0.5*width()*zoomFactor(), h=0.5*height()*zoomFactor(); 
+    if (abs(abs(x-this->x())-w) < portRadiusMult*zoomFactor() &&	  
+            abs(abs(y-this->y())-h) < portRadiusMult*zoomFactor() &&	  
+            abs(hypot((x-this->x()),(y-this->y()))-hypot(w,h)) < portRadiusMult*zoomFactor())	  
```

Comment:
Why is the hypotenuse calculation required?

Reply:
I copied it from clickType() in sheet.cc. I assumed that it is somehow needed to test for clicktype on resize inside all four corners of the bounding box.

Reply:
Fair enough, but it looks redundant to me. Probably my mistake, for being overzealous.

Reply:
I thought it dealt with off diagonal cases.